Pipelines, best practices and filtering
1. [Best practices for variant calling in clinical sequencing](https://genomemedicine.biomedcentral.com/articles/10.1186/s13073-020-00791-w)
2.  [Accurate, scalable cohort variant calls using DeepVariant and GLnexus](https://academic.oup.com/bioinformatics/article/36/24/5582/6064144) 
3.  [Variant interpretation using population databases: Lessons from gnomAD](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC9160216/#:~:text=Variants%20reported%20in%20gnomAD%20have,the%20variant%20and%20the%20site.) 
4. [ Effective variant filtering and expected candidate variant yield in studies of rare human disease](https://www.nature.com/articles/s41525-021-00227-3)
5. [ ForestQC: Quality control on genetic variants from next-generation sequencing data using random forest](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6938691/)
6. [Best practices for evaluating single nucleotide variant calling methods for microbial genomics](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4493402/)
7. [# On the concepts and measures of diversity in the genomics era](https://www.sciencedirect.com/science/article/pii/S2214662823000075) - HWE, etc.

## METHODS

(deprecated?) GATK tutorial to Evaluate a callset - Varianteval - https://sites.google.com/a/broadinstitute.org/legacy-gatk-documentation/tutorials/6211-howto-Evaluate-a-callset-with-VariantEval 

UCLA tutorial - evaluate variant callset with GATK tools - https://qcb.ucla.edu/wp-content/uploads/sites/14/2016/03/IntroductiontoVariantCallsetEvaluationandFilteringTutorialAppendix-LA2016.pdf 

GATK CollectVariantCallingMetrics - https://github.com/broadgsa/gatk/blob/master/doc_archive/tutorials/(howto)_Evaluate_a_callset_with_CollectVariantCallingMetrics.md 